### ISSUE TITLE
[bug] ScrollView - Unable to scrollTo using ref 

### ISSUE THREAD

**Issue by wmonecke** [tamagui/tamagui#823]
**Describe the bug**
I am unable to use the scrollTo method form RN scroll view:

```javascript
export function Container({
  items,
  children,
}: PropsWithChildren & { items: Props }) {
  const scrollView = useRef<ScrollViewProps>();

  const scrollTo = useCallback((y: number) => {
    // eslint-disable-next-line no-console
    console.log('should scroll to', y);

    console.log('scrollView.current', scrollView.current);

    scrollView.current?.scrollTo({ y }); // scrollTo is undefined
  }, []);

  const selectedItem = useMemo(
    () => items.find((item) => item.isSelected),
    [items],
  );

  return (
    <Context.Provider value={{ scrollTo }}>
      <YStack fullscreen bc="$gray2Dark">
        <Header title={selectedItem?.text || 'Not found'} />

        <ScrollView flexGrow={1} ref={scrollView} alwaysBounceVertical>
          {children}
        </ScrollView>

        <BottomNavigationBar items={items} />
      </YStack>
    </Context.Provider>
  );
}
```

**Comment by VladyslavKochetkov** [comment]
Are you sure that the functionality is broken or is it just the typed? I tested on my end and the functionality seems to work.

**Comment by jdordoigne** [comment]
Have you enable react-native-web-lite ?
I can reproduced it when react-native-web-lite is turned on.

**Comment by natew** [comment]
Oh so is this only with react-native-web-lite? I don't see it in the sandbox with `useReactNativeWebLite: true`

**Comment by jdordoigne** [comment]
If I enable `useReactNativeWebLite: true`
I have this error : 
`TypeError: Cannot read properties of undefined (reading 'scrollResponderScrollTo')`

If we go deeper it leads to :
```
this.scrollTo = (y, x, animated) => {
    if (typeof y === "number") {
      console.warn(
        "`scrollTo(y, x, animated)` is deprecated. Use `scrollTo({x: 5, y: 5, animated: true})` instead."
      );
    } else {
      ;
      ({ x, y, animated } = y || emptyObject);
    }
    this.getScrollResponder().scrollResponderScrollTo({
      x: x || 0,
      y: y || 0,
      animated: animated !== false
    });
  };
```

where `this.getScrollResponder()` is undefined

**Comment by natew** [comment]
Ah I found it, this was due to me enabling concurrent mode by removing the class mixin stuff, fix should be in 1.9.4 going out shortly.
